### PR TITLE
Set Design 2

### PR DESCRIPTION
**Pull request by Mehdi-Rh**
In this PR I made the design for the calculator of the web application.
Nb : don't forget to run 

> npm i react-textfit

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,28 +1,9 @@
-import logo from './logo.svg';
-import './App.css';
+import React from 'react';
+import Calculator from './components/Calculator';
 
 function App() {
```
Comment (mmsesay) [inline src/App.js:4]: - [ ] For this task, the `App.js` should be class-based, not functional-based. Let's try to update it.
See example:
```js
class App extends Component {
  render() {
    // reference as a js object
    return {
      <Calculator />
    };
  }
}
```

File: src/components/Calculator.js [hunk 2]
```diff
@@ -0,0 +1,65 @@
+import React, { Component } from 'react';
+import { Textfit } from 'react-textfit';
```
Comment (mmsesay) [inline src/components/Calculator.js:2]: - [ ] I got a compilation error when I try to start the app. Can you please take a look at it? It seems like the problem is coming from `react-textfit`. 

Screenshot |
:--------------:|
a1

**Review by mmsesay — CHANGES_REQUESTED** [review]
Hi @Mehdi-Rh,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/App.js (hunk 1 above)
Comment (Mehdi-Rh) [reply src/App.js:4]: I have done this but linter for App.js forced me to make it a pure function!

File: src/components/Calculator.js (hunk 2 above)
Comment (Mehdi-Rh) [reply src/components/Calculator.js:2]: Yes it's ok, I added in the PR description the command to run:
Nb: don't forget to run
> npm i react-textfit

Comment (Amrhub) [reply src/components/Calculator.js:2]: @Mehdi-Rh I highly suggest to add it to package.json

**Review by Amrhub — APPROVED** [review]
# Approved 💯 ✅ 🥳 🎈 🎉 

Hi @Mehdi-Rh, 👋🏻 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights ✅ 
Highly organized and clean code ✔️ 👨🏻‍💻 
Professional documentation ✔️ 
Correct workflow ✔️ 
Meaningful commit messages ✔️ 



## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- [ ] [OPTIONAL] add react-textfit to package.json

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**